tests/TwigMiddlewareTest.php

```diff
@@ -9,207 +9,92 @@
 
 namespace Slim\Tests;
 
-use DI\Container;
-use Prophecy\Argument;
-use Prophecy\Prophecy\ObjectProphecy;
 use Psr\Container\ContainerInterface;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 use Psr\Http\Message\UriInterface;
 use Psr\Http\Server\RequestHandlerInterface;
-use ReflectionProperty;
-use RuntimeException;
 use Slim\App;
```

Comment:
Please import the `RuntimeException` right before the `Slim\App`, as there are docblocks that use the `RuntimeException`.
```php
    /**
     * @expectedException RuntimeException
     * @expectedExceptionMessage The app does not have a container.
     */
```

Reply:
Done